app/services/audited_operations_service.rb

```diff
@@ -7,25 +7,14 @@ class ShareRevokedError < StandardError
   class UnauthorisedAccessRequestError < StandardError
   end
 
-  # constants for User/BirthRecord actions
-  ADD_BIRTH_RECORD_TO_USER = 'ADD_BIRTH_RECORD_TO_USER'
-  REMOVE_BIRTH_RECORD_FROM_USER = 'REMOVE_BIRTH_RECORD_FROM_USER'
-
-  # constants for User/Share actions
-  SHARE_BIRTH_RECORD = 'SHARE_BIRTH_RECORD'
-  REVOKE_SHARE = 'REVOKE_SHARE'
-
-  # constants for Recipient/Share actions
-  VIEW_SHARED_BIRTH_RECORD = 'VIEW_SHARED_BIRTH_RECORD'
-
   def self.add_birth_record_to_user(user:, birth_record:)
     raise ArgumentError, 'user cannot be nil' if user.nil?
 
     Audited.audit_class.as_user(user) do
-      user.birth_records_users << BirthRecordsUser.create!(
+      user.user_documents << UserDocument.create!(
```

Comment:
i... ponder if they could be called user.documents...

Reply:
What do you mean?